src/webots/nodes/utils/WbUrl.cpp

```diff
@@ -119,6 +120,14 @@ QString WbUrl::computePath(const WbNode *node, const QString &field, const QStri
     if (QFileInfo(path).exists())
       return checkIsFile(node, field, path);
     const QString error = QObject::tr("'%1' not found.").arg(url);
+
+    if (isLocalUrl(url)) {
+      QString newUrl(url);
+
+      newUrl.replace("webots://", "https://raw.githubusercontent.com/" + WbApplicationInfo::repo() + "/" +
+                                    WbApplicationInfo::branch() + "/");
+      return newUrl;
+    }
```

Comment:
It seems better to move down the declaration of the error string:
```suggestion

    if (isLocalUrl(url)) {
      QString newUrl(url);
      newUrl.replace("webots://", "https://raw.githubusercontent.com/" + WbApplicationInfo::repo() + "/" +
                                    WbApplicationInfo::branch() + "/");
      return newUrl;
    }
    
    const QString error = QObject::tr("'%1' not found.").arg(url);
```